site/content/api/Event-API.md

```diff
@@ -77,3 +82,31 @@ The following sequence diagram illustrates the flow of messages involved in a *B
 **Message Format**
 
 See [*Telemetry API*]({{< relref "Telemetry-API.md" >}}) for definition of message format. 
+
+## Well-known event message types
+
+Hono defines *well-known* events that are of a specific *content-type*.
+
+In the following these events (currently one) are specified in detail.
+
+
+### Device command readiness notification
+
+An event of this type does not have any payload so the body of the event MUST be empty (different from other messages sent downstream).
+It only carries AMQP 1.0 properties.
+
+The AMQP 1.0 properties an event sender needs to set for a *device command readiness notification* event (as defined in the [*Telemetry API*]({{< relref "Telemetry-API.md" >}})) are defined in the following table:
+
+| Name           | Mandatory | Location                 | Type      | Description |
+| :------------- | :-------: | :----------------------- | :-------- | :---------- |
+| *content-type* | yes       | *properties*             | *symbol*  | Must be set to *application/vnd.eclipse-hono-empty-notification* |
+| *ttd*          | yes       | *application-properties* | *int*     | 'time til disconnect' : see [*Telemetry API*]({{< relref "Telemetry-API.md" >}}). | 
+
+Note that for this event type the property `ttd` must be set, as otherwise a receiver could not determine if the event is still valid when it receives it (different from other messages sent downstream).
+
+
+NB: Such an event is typically issued by the device itself to indicate that it is ready to receive a command. In future releases the protocol adapters may be able to send this
```

Comment:
this should be rephrased to give an example of what an *empty notification* could be used for.